data.sql

```diff
@@ -3,3 +3,8 @@
 INSERT INTO animals (name) VALUES ('Luna');
 INSERT INTO animals (name) VALUES ('Daisy');
 INSERT INTO animals (name) VALUES ('Charlie');
```

Comment:
Kindly remove the code which is not relevant to this project to keep your repo clean.